api-tester/src/defaults/kotlin/com/revenuecat/apitester/kotlin/revenuecatui/PaywallFooterViewAPI.kt

```diff
@@ -14,7 +15,7 @@ import com.revenuecat.purchases.ui.revenuecatui.views.PaywallFooterView
 private class PaywallFooterViewAPI {
 
     fun checkType(context: Context) {
-        val paywallFooterView: FrameLayout = PaywallFooterView(context)
+        val paywallFooterView: AbstractComposeView = PaywallFooterView(context)
```

Comment:
This is a breaking change... But feels that it should be ok... Any thoughts?

Reply:
If it works I think we should go for it, it will simplify the views

Reply:
Paywalls is still experimental so :)